dts: remove unused clock-controller property

galak opened the pull request:
The 'clock-controller' property isn't specified and not used for any
generation.  Lets remove it.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

galak committed:
dts: remove unused clock-controller property

The 'clock-controller' property isn't specified and not used for any
generation.  Lets remove it.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

erwango commented:
That's fixes the problem.
You might want to remove the STM32 "clocks-controller" as well

galak committed:
dts: remove unused clock-controllers property

The 'clock-controllers' property isn't specified and not used for any
generation.  Lets remove it.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

erwango commented:
To be reported in :
- [x] https://github.com/zephyrproject-rtos/zephyr/pull/17100 (STM32H7)
- [ ] https://github.com/zephyrproject-rtos/zephyr/pull/16962 (STM32G0)
